openstudio-geb.gemspec

```
@@ -22,11 +22,14 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
-  spec.add_development_dependency 'bundler', '~> 2.1'
+  spec.required_ruby_version = '~> 3.2.2'
```

```suggestion
  spec.required_ruby_version = '3.2.2'
```

We support exactly Ruby v3.2.2 because os-extension-gem requires bundler==2.4.10 and that requires Ruby 3.2.2: https://stdgems.org/bundler/